docs/tutorials/own_machine/04_clone_renku_project.rst

```diff
@@ -0,0 +1,90 @@
+.. _clone_renku_project:
+
+Clone your Renku project
+========================
+
+In order to save your work back to RenkuLab from your own machine,
+you'll need to create an SSH key and save that key in your Renku GitLab account.
+
+If you already have an SSH key in your RenkuLab GitLab profile, then skip down to "Clone your Renku Project" below.
+Otherwise, follow along to set up a key.
+
+Enable remote access to your Renku Project via an SSH Key
```

Comment:
you don't need to do this - use `renku login` instead. Also, it's recommended to use the `renku clone` command rather than `git clone` because it will set up all the git hooks and other repo settings right away.

Reply:
re `renku login`: It seems like this depends on a browser. Would this work if renku is installed on a remote server where the user is ssh'ed?

Reply:
yes, you're right, it depends on the browser. I assumed if you are running interactive sessions there is a browser involved, but there might certainly be cases where opening one up directly is not an option. @mohammad-sdsc is there any way to make that work from a remote system?

Reply:
I suppose we could offer a page somewhere that gives the user a token to copy/paste into the terminal if the browser flow is not possible

Reply:
For some additional context- A common workflow I have seen (and personally used) when working on data science projects is to ssh into a remote machine with a port opened specifically for Jupyter. In this case, the (local) browser would not be available to renku installed on the remote machine.

Reply:
FYI in the new "clone" section, I added instructions for both the `renku login` method and ssh key method, depending on what kind of environment the user is in.